Quadlet Container: Add support for EnvironmentFile and EnvironmentHost (#16953)
containers/podman: docs/source/markdown/podman-systemd.unit.5.md

[services in systemd](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Environment=)
and can be listed multiple times.

#### `EnvironmentFile=`

> Use a line-delimited file to set enviroment variabled in the container.

Reviewer: ```suggestion
Use a line-delimited file to set environment variables in the container.
```

The path may be absolute or relative to the location of the unit file.

> This key may be used multiple times and the order to persist when passed to `podman run`.

Reviewer: ```suggestion
This key may be used multiple times, and the order persists when passed to `podman run`.
```

#### `EnvironmentHost=` (defaults to `no`)

> Use  host  environment  inside  of the container.

Reviewer: ```suggestion
Use the host environment inside of the container.
```
